modules/custom/az_paragraphs/az_paragraphs_cards/config/install/field.field.paragraph.az_cards.field_az_title.yml

```
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_az_title
+    - paragraphs.paragraphs_type.az_cards
+id: paragraph.az_cards.field_az_title
+field_name: field_az_title
+entity_type: paragraph
+bundle: az_cards
+label: Title
```

Suggestion from 2023-02-01 PR Meeting:
```suggestion
label: Card group title
```